docs/content/docs/deployment/config.md

```diff
@@ -34,6 +34,8 @@ The configuration is parsed and evaluated when the Flink processes are started.
 
 The out of the box configuration will use your default Java installation. You can manually set the environment variable `JAVA_HOME` or the configuration key `env.java.home` in `conf/flink-conf.yaml` if you want to manually override the Java runtime to use.
 
+You can specifiy a different configuration directory location by defining the `FLINK_CONF_DIR` environment variable. For resource-providers which provide non-session deployments, you can specifiy per-job configurations this way. Make a copy of the `conf` directory from the Flink distribution and modify the settings on a per-job basis. Note that this is not supported in Docker or standalone Kubernetes deployments.
+
```

Comment:
Maybe also mention `FLINK_PROPERTIES` as the alternative in docker/standalone-k8s deployments. This, instead of specifying a different configuration directory, appends given configurations to the flink-conf file in the image.
https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/deployment/resource-providers/standalone/docker/